for-users/software/ddl.rst

```diff
@@ -0,0 +1,262 @@
+
+*************************************************************************************
+Using IBM Watson Machine Learning Community Edition on the ORNL Summit Super Computer
```

Comment:
This should be short enough to fit ideally on one line in the sidebar navigation. Suggest `IBM Watson Machine Learning CE`

Reply:
I changed it to `IBM Watson Machine Learning CE` and it still line wraps on my screen. 
What do you think about:

- `IBM Watson ML CE`
- `Watson Machine Learning CE`